charts/external-dns/templates/deployment.yaml

```diff
@@ -132,6 +136,47 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+        {{- if eq $provider "webhook" }}
+        {{- with .Values.provider }}
+        - name: {{ $providername }}
```

Comment:
It would simplify administration if the webhook sidecar had a fixed name.

Reply:
I don't have a strong opinion on this one. 
It's also a good known administration practice to name things precisely.
@stevehipwell @mrueg Wdyt ?

Reply:
I agree that using a fixed name for the container would be preferable.

Reply:
Implemented with 91095975